R/qc_sequence_coverage.R

```diff
@@ -47,6 +47,12 @@ qc_sequence_coverage <- function(data,
                                  coverage,
                                  sample = NULL,
                                  interactive = FALSE) {
+
+  # Validate inputs
+  if (!all(c({{ protein_identifier }},{{ coverage }}) %in% colnames(data))) {
```

Comment:
```suggestion
  if (!all(c([{{ protein_identifier }}](rlang::as_name(rlang::enquo(protein_identifier))),[{{ coverage }}](rlang::as_name(rlang::enquo(coverage)))) %in% colnames(data))) {
```

This should be the right syntax